uni/lib/controller/local_storage/app_restaurant_database.dart

```diff
@@ -111,3 +110,27 @@ class RestaurantDatabase extends AppDatabase {
     await txn.delete('restaurants');
   }
 }
+
+List<Restaurant> hideMeals(List<Restaurant> restaurants) {
```

Comment:
What about `filterCurrentWeekMeals`?

Reply:
Sounds better. Will change.

Reply:
But semantically is a bit different... I will leave it as `filterPastMeals`